exercises/list-ops/canonical-data.json

```diff
@@ -40,7 +40,7 @@
       ]
     },
     {
-      "description": "concat lists and lists of list into a new list",
+      "description": "given a list of lists, concatenate each list to the previous list",
```

Comment:
**nitpick**: "concatenate ... to" feels awkward to me, as does "the previous list."

Perhaps something more like the following would be better.
> concatenate a list of lists